src/FakeItEasy.Analyzer/UnusedReturnValueAnalyzer.cs

```diff
@@ -42,13 +39,8 @@ private static void AnalyzeCall(SyntaxNodeAnalysisContext context)
                 return;
             }
 
-            var diagnosticName = GetDiagnosticName(methodSymbol);
-            if (diagnosticName == null)
-            {
-                return;
-            }
-
-            var descriptor = Diagnostics.GetValueOrDefault(diagnosticName);
+            string methodFullName = $"{methodSymbol.ContainingType.GetFullName()}.{methodSymbol.GetDecoratedName()}";

```

Comment:
Sorry, sorry :raised_hands: 

I won't insist on it, I just couldn't resist :wink: 

Actually, I think it will trigger a stylecop warning. Normally I'd take a look at the build log, but we're still waiting for @OrchardCMS's [behemoth](https://twitter.com/adamralph/status/733572470440001537) to finish....